spec/Appendix C -- Schema Coordinates.md

```diff
@@ -0,0 +1,93 @@
+# Appendix: Schema Coordinates
```

Comment:
Rather than an additional Appendix, might this reasonably fit as section 3.3.3 after Schema Extensions?

Reply:
ack - had it as a separate thing to err on the side of not wanting to confuse/interfere with the actual runtime language syntax - but makes sense to me! moved as suggested.